### ISSUE TITLE
npm audit shows 2 high severity vulnerabilities

### ISSUE THREAD

**Issue by neelkarma** [gatsbyjs/gatsby#28858]
## Description

When running `npm audit` on a brand new Gatsby project, it shows 2 high severity vulnerabilities relating to a package called `axios`. Here's the output:
```
                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  High            Server-Side Request Forgery

  Package         axios

  Patched in      >=0.21.1

  Dependency of   gatsby

  Path            gatsby > axios

  More info       https://npmjs.com/advisories/1594


  High            Server-Side Request Forgery

  Package         axios

  Patched in      >=0.21.1

  Dependency of   gatsby

  Path            gatsby > gatsby-cli > gatsby-recipes > contentful-management
                  > axios

  More info       https://npmjs.com/advisories/1594

found 2 high severity vulnerabilities in 1892 scanned packages
  2 vulnerabilities require manual review. See the full report for details.
```

### Steps to reproduce

1. Run `gatsby new <your name here> https://github.com/gatsbyjs/gatsby-starter-hello-world`
2. Run `cd <your name here>`
3. Run `npm audit fix`
4. Run `npm audit`

Reproduction: https://github.com/neelkarma/gatsby-bug-audit-axios

### Expected result

When running `npm audit`, there should be no vulnerabilities (or at least vulnerabilities that can be fixed with `npm audit fix`)

### Actual result

When running `npm audit`, 2 high severity vulnerabilities are listed, requiring manual review to fix. They both seem to be connected to the package `axios` (see description above for the output)

### Environment

```
  System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 AMD Ryzen 7 3700U with Radeon Vega Mobile Gfx  
  Binaries:
    Node: 12.18.3 - C:\Program Files\nodejs\node.EXE
    npm: 6.14.6 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 3.9.0
  Browsers:
    Chrome: 87.0.4280.88
    Edge: Spartan (44.19041.423.0), Chromium (87.0.664.66)
  npmPackages:
    gatsby: ^2.26.1 => 2.26.1 
  npmGlobalPackages:
    gatsby-cli: 2.16.2
```

**Comment by fabiosanto** [comment]
same here 
<img width="819" alt="Screen Shot 2021-01-06 at 7 24 26 am" src="https://user-images.githubusercontent.com/879072/103700467-37b0a180-4ff0-11eb-80e0-df30f9c56c17.png">

**Comment by KhizerRehan** [comment]
`npm audit`

┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Server-Side Request Forgery                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ axios                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.21.1                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ gatsby                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ gatsby > gatsby-cli > gatsby-recipes > contentful-management │
│               │ > axios                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1594

**Comment by neelkarma** [comment]
@KhizerRehan the pr isn't merged yet, but it looks like it's updating axios, which is good.
